ammonite-docker/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM openjdk:8-jdk-alpine
```

Comment:
I'd be cool to switch the Marathon Docker image to Alpine as well :).

Reply:
Unfortunately, the native mesos lib depends on glibc and does not work with Alpine's libc :( I'm not sure if it is possible to make it work.